### PR TITLE
Simplify and clean up test exercise resources

### PR DESCRIPTION
**Pull request by a544jh**
Issue #75

### CODE REVIEW

**Comment by a544jh** [comment]
I've cleaned up the courses json and testRescources directory, and renamed the courses to something more generic. However, exercise names remain unchanged.

File: src/test/java/fi/helsinki/cs/tmc/core/commands/DownloadExercisesTest.java [hunk 1]
```diff
@@ -53,6 +53,9 @@
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
```
Comment (jamo) [inline src/test/java/fi/helsinki/cs/tmc/core/commands/DownloadExercisesTest.java]: Incorrect order of imports,  the import order and grouping should be:
- All static imports in a single group
- fi.helsinki.cs.tmc imports (only if this source file is in the fi.helsinki.cs.tmc package space)
- Third-party imports, one group per top-level package, in ASCII sort order
- for example: android, com, junit, org, sun
- java imports
- javax imports

**Comment by jamo** [comment]
Nice work; I really like the cleanup of the unnecessary files and the generification of the filenames.
There is clearly still some work to the optimal end result, though :)

Should we merge this in a few chunks, current state of this being one of them, to reduce the size of possible merge conflicts.

Prior to merge you need to rebase this against our current master to make this mergeable.

**Comment by a544jh** [comment]
Rebased & tests fixed

**Comment by ljleppan** [comment]
LGTM

**Pull request merged**